src/sender.js

```diff
@@ -0,0 +1,114 @@
+require('dotenv').config()
+const path = require('path')
+const Web3 = require('web3')
+const { connectSenderToQueue } = require('./services/amqpClient')
+const { redis, redlock } = require('./services/redisClient')
+const { getGasPrices } = require('./services/gasPrice')
+const { sendTx, sendRawTx } = require('./tx/sendTx')
+const { getNonce } = require('./tx/web3')
+const { syncForEach } = require('./utils/utils')
+
+const { VALIDATOR_ADDRESS, VALIDATOR_ADDRESS_PRIVATE_KEY } = process.env
+
+const config = require(path.join('../config/', process.argv[2]))
+
+const provider = new Web3.providers.HttpProvider(config.url)
+const web3Instance = new Web3(provider)
+const nonceLock = `lock:${config.id}:nonce`
+const nonceKey = `${config.id}:nonce`
+let chainId = 0
+
+async function initialize() {
+  chainId = await sendRawTx({
```

Comment:
Here we can use `web3.eth.net.getId()`, unless there is a good reason to use raw HTTP.